core/class-kirki-init.php

```diff
@@ -316,4 +328,91 @@ public function remove_controls( $wp_customize ) {
 			$wp_customize->remove_control( $control );
 		}
 	}
+
+	/**
+	 * Shows an admin notice.
+	 *
+	 * @access public
+	 * @since 3.0.42
+	 * @return void
+	 */
+	public function admin_notices() {
+
+		// No need for a nag if we don't need to recommed installing the FA plugin.
+		if ( ! self::$show_fa_nag ) {
+			return;
+		}
+
+		// No need for a nag if FA plugin is already installed.
+		if ( defined( 'FONTAWESOME_DIR_PATH' ) ) {
+			return;
+		}
+
+		// No need for a nag if current user can't install plugins.
+		if ( ! current_user_can( 'install_plugins' ) ) {
+			return;
+		}
+
+		// No need for a nag if user has dismissed it.
+		$dismissed = get_user_meta( get_current_user_id(), 'kirki_fa_nag_dismissed', true );
+		if ( true === $dismissed || 1 === $dismissed || '1' === $dismissed ) {
+			return;
+		}
+
+		?>
+		<div class="notice notice-info is-dismissible">
+			<p>
+				<?php esc_html_e( 'Your theme uses a font-awesome field for icons. To avoid issues with missing icons on your frontend we recommend you install the Font Awesome plugin.', 'kirki' ); ?>
```

Comment:
very minor: the control was fontawesome, not font-awesome. But since this is user facing 'Font Awesome' may even be better as that's the official branding.

Reply:
Good point, fixed